package excecao.personalizadaA;

import excecao.Aluno;

public class TesteValidacoes {

	
	public static void main(String[] args) {
		try {
			Aluno aluno = new Aluno ("", -7);
			
			Validar.aluno(aluno);
		} catch (Exception e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}
		
		System.out.println("FIM :)");
	}
}
